src/components/hooks/useStream.js

```diff
@@ -3,24 +3,24 @@ import {useEffect, useRef} from 'react';
 /**
  * Custom hook that returns a video reference with given stream object attached to it.
  *
- * @param {MediaStream} stream  Media stream instance to pass to the video reference.
+ * @param {MediaStream} stream  Media stream instance to pass to the audio/video reference.
  * @returns {Element} Video element reference
  */
-export default function useVideo(stream) {
-  const videoRef = useRef({});
+export default function useStream(stream) {
+  const streamRef = useRef({});
 
   useEffect(() => {
-    const video = videoRef.current;
+    const av = streamRef.current;
```

Comment:
I think calling it `mediaElement` would be more descriptive
```suggestion
    const mediaElement = streamRef.current;
```